### PR TITLE
[ZEPPELIN-1644] make document easier to follow key instructions

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?
Doc should deliver key features and recommended usage more simple and easy way.

 - docs/install/install.md has lots of duplicated section with README.md.
 - docs/install/install.md includes install from binary as well as build from source. I've seen that makes some beginners try download binary and then source build it again.
 - recommended and key usage need to be highlighted.
 - Be less verbose in key instructions. Move optional, additional info from in the middle of key instruction to end of the each page.

### What type of PR is it?
Improvement

### Todos
* [x] - improve doc

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1644

### How should this be tested?
Run doc locally

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
cc @AhyoungRyu

File: README.md [hunk 1]
```diff
@@ -17,323 +17,13 @@ Core feature:
...
 
-```
-sudo apt-get update
-sudo apt-get install git
-sudo apt-get install openjdk-7-jdk
-sudo apt-get install npm
-sudo apt-get install libfontconfig
-```
+### Build from source
+Please check [Build from source](http://zeppelin.apache.org/docs/snapshot/install/build.html) to build Zeppelin from source.
```
Comment (AhyoungRyu) [inline README.md]: I can't find `build.md` file. Is this intended?

**Comment by tae-jun** [comment]
@Leemoonsoo Nice! When I was at Zeppelin meetup, every beginner tried to build Zeppelin. Because they didn't know about existence of Zeppelin binary :-)

So I suggest adding a link to http://zeppelin.apache.org/download.html on `README.md` instead of [install](http://zeppelin.apache.org/docs/snapshot/install/install.html), and make super big button for downloading `Binary package with all interpreters` on `Download.html`. Too many options make beginners confused 😄

What do you think?

And like @AhyoungRyu said above, [install](http://zeppelin.apache.org/docs/snapshot/install/install.html) is invalid.

**Comment by AhyoungRyu** [comment]
@leemoonsoo I've always thought that such long README.md makes the entrance barrier high so that beginner can't start Zepplin easily. Definietly better tha before. @tae-jun maybe the invaild snapshot link is related to #1616. Am i right?

**Comment by Leemoonsoo** [comment]
Pushed missing `build.md`.
Thanks @tae-jun @AhyoungRyu for the review.

I agree on improving `download` page, make sure using binary package as a first option, build from source is optional / or for development. Thanks for suggestion @tae-jun. While `download` page resides on `gh-pages` branch, i'll make a separate pullrequest after merge this PR.

**Comment by jfanegan** [comment]
As a newbie to Zeppelin and this being my first contribution (by the way,
I'm still unable to get Zeppelin installation on Window 7 to work properly.
When I start Zeppelin from Git or Cygwin, I get this frustrating, annoying
output: zeppelin process died [failed], and in the log file, Error: Could
not find or load main class org.apache.zeppelin.server.ZeppelinServer),

I'll suggest separating installation steps and requirements by OS
environment - both for binary package installation and for building from
source. There are issues with installing Zeppelin on Windows, hence the
reason why this guy built this product:
http://mund-consulting.com/Products/Sparklet.aspx. However, his product
doesn't have the latest release or all the interpreters installed - reason
why I'm still trying the installation.

It'd be nice to have beginner's videos on installing in each OS. I've seen
a video for Ubuntu installation, but none for Windows. None of the folks
that I know have been able to get this working on Window 7.

I don't know if this is right place for my comments, but apologies if it is
not..

**Comment by AhyoungRyu** [comment]
@Leemoonsoo There is one more good place to add the location of `build.md` : [docs/0.7.0-SNAPSHOT/#quick-start](https://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/#quick-start). And LGTM :)

**Comment by AhyoungRyu** [comment]
@jfanegan Thanks for sharing your issue and considerate feedback as well. Yeah it makes sense. I've also seen many Windows users struggling with Zeppelin installation with various error cases. 
How about creating a new Jira ticket for this?  You can create it in here: [https://issues.apache.org/jira/browse/zeppelin](https://issues.apache.org/jira/browse/zeppelin). Then the title can be "Separating installation steps and requirements by OS environment" as you said.

**Comment by Leemoonsoo** [comment]
@jfanegan Thanks for the feedback and suggestion. We can address that in the other issue.

@AhyoungRyu @tae-jun Thanks for the review and merge to mater if there're no further comments.

**Pull request closed without merging**